test/unit/config/article_scrape_patterns_test.rb

```diff
@@ -57,7 +57,9 @@ def test_scrape_methods_presets_are_valid
   end
 
   def test_scrape_methods_have_proper_variable_names
-    article_scrape_patterns = File.read(NewsScraper::Constants::SCRAPE_PATTERN_FILEPATH).split("\n").map(&:strip)
+    article_scrape_patterns = File.read(NewsScraper::Configuration::DEFAULT_SCRAPE_PATTERNS_FILEPATH)
```

Comment:
Can't do that. This test is asserting that the assumed variable names are correct. e.g. `&class_author`